### ISSUE TITLE
authenticated user field ignored

### ISSUE THREAD

**Issue by trentfisher** [allinurl/goaccess#516]
For my web servers, almost all accesses are authenticated, so the third field (apache common format) has meaningful data which I'd like to get metrics on.  But it appears you ignore that field.  Any chance of getting that added?

Thanks!

**Comment by allinurl** [comment]
I thought about the userid part. I can certainly have a panel dedicated to authenticated users. Thanks for suggesting this!

Stay tuned!

**Comment by huayra** [comment]
Thanks for goaccess, it feels like the webalizer of this decade :+1: 

While testing it I came across the lack of support for the authenticated user field. Being able to filter by username and see that in the output would be a great addition!

Keep up the good work Gerardo! :-)

**Comment by allinurl** [comment]
@huayra , thanks for the feedback. 

I'm close to push the upcoming version so I plan to have the authenticated user field either in this version or the one after. As far as filtering, it's definitely coming, issue **#117** will address this,  it will be the next big update that GoAccess will get, both in the HTML side and the terminal side.

**Comment by huayra** [comment]
@allinurl thanks for your quick reply!

If there is a certain branch (or just master) you want testing on I'll be happy to try. Thrilled to hear that filtering is also in the works as well :-)

Keep up the good work!

**Comment by allinurl** [comment]
This has been implemented (commit: [**1c2750b**](https://github.com/allinurl/goaccess/commit/1c2750ba0419a79b71642ac29956f1f4028a2bb3)). It will be deployed in the upcoming release (pretty soon).

Feel free to build from [**development**](https://github.com/allinurl/goaccess#build-from-github-development) if needed.

**Comment by huayra** [comment]
Thank you Gerardo!

Built and tested both in the ncurses client and the HTML output. It works with %e% as announced in the commit.

I have to say that the changes you have made to the HTML output are amazing, people should be really looking forward to the next release :-)

The only caveat I had was on me: figuring out how to avoid the user field being "-" instead of "authenticatedusername(s)" but after some trials I manged to get it working.

In case this is useful for anybody I changed the log format field in the goaccess.conf file as follows:
> # NCSA Combined Log Format
> log-format %h %- %e %^[%d:%t %^] "%r" %s %b "%R" "%u"

In the meantime I'm going to make some nice dashboards while I patiently wait for the filtering :-D I'll follow #117
